.github/workflows/release.yaml

```diff
@@ -1,5 +1,9 @@
 on:
   push:
+    branches:
+      - main
```

Comment:
Actually, the `release` pipeline is currently the only one to build rustic for all supported targets. If we introduce some platform-specific thing that breaks a specific target, it would fail in this pipeline. Therefore I would remove the "push-on-main-only" or replace it with "on pull-request-to-main"

Reply:
Ah, yes. You are right!